### PR TITLE
Feature/add marine biodiversity layer

### PR DESCRIPTION
**Pull request by weberjavi**
### Description
This PR adds the tile layers for _fishes priority_, _marine mammals priority, rarity and richness_, and _marine vertebrates priority, rarity and richness_.
It also adds a documentation page, under the `docs` folder, that explains the steps to follow in order to add new layers to the project.
Some changes to file names have been made.
### Testing instructions
Check the biodiversity sidebar and toggle  marine layers. Make sure the metadata for the layers is there after clicking the info buttons.
Read the `docs` file `layers-update-flow.md`.
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-20

### CODE REVIEW

File: docs/layers-update-flow.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    ```js
+    [HUMMINGBIRDS_RARITY]: {
+      title: HUMMINGBIRDS_RARITY,
+      slug: HUMMINGBIRDS_RARITY,
+      type: LAYER_TYPES.TILE_LAYER,
+      url: LAYERS_URLS[HUMMINGBIRDS_RARITY],
+      bbox: [-164,-40,-35,56]
+    }
+    ```
+- The last step for a layer to be ready to be usede on the platform is to add the needed configuration to be consumed by the `legend` component. Diferent legend configuration are created depending on the category of the layer (_biodiversity, _human pressures_ or _protected areas_), and those are stored into the associated file in the `constants` folder. In this case, fishes priority layer, the legend config to be updated is located on the `constants/biodiversity-layers-constants` file.
```
Comment (gretacv) [inline docs/layers-update-flow.md:58]:
```suggestion
- The last step for a layer to be ready to be used on the platform is to add the needed configuration to be consumed by the `legend` component. Different legend configurations are created depending on the category of the layer (_biodiversity_, _human pressures_ or _protected areas_), and those are stored into the associated file in the `constants` folder. In this case, fishes priority layer, the legend config to be updated is located on the `constants/biodiversity-layers-constants` file.
```

File: src/constants/biodiversity-layers-constants.js [hunk 2]
```diff
@@ -1,5 +1,589 @@
...
+    ...defaultGradientConfig,
+    title: "Fishes rarity"
+  },
+  [FISHES_RICHNESS]: {
+    ...defaultGradientConfig,
+    title: "Fishes richness"
+  },
+  [FISHES_PRIORITY]: {
+    ...defaultGradientConfig,
+    title: "Fishes priority"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:494]:
```suggestion
    title: "Global fishes priority"
```

File: src/constants/biodiversity-layers-constants.js [hunk 3]
```diff
@@ -1,5 +1,589 @@
...
+    ...defaultGradientConfig,
+    title: "Marine mammals richness"
+  },
+  [MARINE_MAMMALS_RARITY]: {
+    ...defaultGradientConfig,
+    title: "Marine mammals rarity"
+  },
+  [MARINE_MAMMALS_PRIORITY]: {
+    ...defaultGradientConfig,
+    title: "Marine mammals priority"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:506]:
```suggestion
    title: "Global marine mammals priority"
```

File: src/constants/biodiversity-layers-constants.js [hunk 4]
```diff
@@ -1,5 +1,589 @@
...
+        }
+      ]
+    },{
+      name: 'MARINE SPECIES',
+      description: 'Global, ~55 km cell size mapping of marine species. ',
+      subcategories: false,
+      taxa: [
+        {
+          value: 'marine vertebrates',
+          name: 'marine vertebrates',
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:95]:
```suggestion
          name: 'all marine vertebrates',
```

File: docs/layers-update-flow.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      FISHES_PRIORITY
+    } from 'constants/layers-slugs';
+
+    export const legendConfigs = {
+      [FISHES_PRIORITY]: {
+        ...defaultGradientConfig,
+        title: "Fishes priority"
+      }
+    }
+    ```
```
Comment (gretacv) [inline docs/layers-update-flow.md:70]:
```suggestion
    ```
- Once the layer is ready to be used by the platform some changes in the UI are needed. Change the size of the sidebar card in `src/components/biodiversity-sidebar-card/biodiversity-sidebar-card-styles.module.scss` so the new element is visible. To have the opening effect the size of the side bar is provided manually.  
```

File: src/constants/biodiversity-layers-constants.js [hunk 6]
```diff
@@ -1,5 +1,589 @@
...
+    title: "Hummingbirds richness"
+  },
+  [HUMMINGBIRDS_RARITY]: {
+    ...defaultGradientConfig,
+    title: "Hummingbirds rarity"
+  },
+  // Global data
+  [MAMMALS_PRIORITY]: {
+    ...defaultGradientConfig,
+    title: "Global Mammals priority"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:474]:
```suggestion
    title: "Global Terrestrial Mammals priority"
```

File: src/constants/biodiversity-layers-constants.js [hunk 7]
```diff
@@ -1,5 +1,589 @@
...
+  },
+  [MAMMALS_RARITY]: {
+    ...defaultGradientConfig,
+    title: "Mammals rarity"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:478]:
```suggestion
    title: "Terrestrial Mammals rarity"
```

File: src/constants/biodiversity-layers-constants.js [hunk 8]
```diff
@@ -1,5 +1,589 @@
...
+  },
+  [MAMMALS_RICHNESS]: {
+    ...defaultGradientConfig,
+    title: "Mammals richness"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:482]:
```suggestion
    title: "Terrestrial Mammals richness"
```

File: src/constants/biodiversity-layers-constants.js [hunk 9]
```diff
@@ -1,5 +1,589 @@
...
+    ...defaultGradientConfig,
+    title: "Birds richness"
+  },
+  [ALL_TAXA_PRIORITY]: {
+    ...defaultGradientConfig,
+    title: "Global terrestrial vertebrate priority"
+  },
+  [ALL_TAXA_RARITY]: {
+    ...defaultGradientConfig,
+    title: "All groups rarity"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:578]:
```suggestion
    title: "All terrestrial groups rarity"
```

File: src/constants/biodiversity-layers-constants.js [hunk 10]
```diff
@@ -1,5 +1,589 @@
...
+  },
+  [ALL_TAXA_RICHNESS]: {
+    ...defaultGradientConfig,
+    title: "All groups richness"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:582]:
```suggestion
    title: "All terrestrial groups richness"
```

File: src/constants/biodiversity-layers-constants.js [hunk 11]
```diff
@@ -1,5 +1,589 @@
...
+    ...defaultGradientConfig,
+    title: "Marine vertebrates richness"
+  },
+  [ALL_MARINE_VERTEBRATES_RARITY]: {
+    ...defaultGradientConfig,
+    title: "Marine vertebrates rarity"
+  },
+  [ALL_MARINE_VERTEBRATES_PRIORITY]: {
+    ...defaultGradientConfig,
+    title: "Marine vertebrates priority"
```
Comment (gretacv) [inline src/constants/biodiversity-layers-constants.js:518]:
```suggestion
    title: "Global marine vertebrates priority"
```

**Review by gretacv — CHANGES_REQUESTED** [review]
Javi, the only bit of code that needs change is the name on the legend for the `global marine vertebrates` priority. 
I also updated the metadata for the text `Where to start to protect?`, I am unsure if you have verified the pixel space is correct following design. However, maybe it is not worth checking it as maybe MOL share an updated text that changes the size of the card.

**Comment by weberjavi** [comment]
@gretacv everything working great :)

Do you approve it?

image

**Comment by weberjavi** [comment]
gonna merge it :D

**Pull request merged**